### PR TITLE
refactor(repo): Autocad/Civil/Asteel solution filter

### PR DESCRIPTION
**Pull request by phliberato**
## Description & motivation

- Civil2023 and AdvanceSteel2023 Converter projects added into solution filter
- Xcopy of Triangle.NET changed from ConverterAdvanceSteel2023
- New ConnectorAdvanceSteel.slnf to filter AS Projects only. (If I need to change Autocad Projects I will open another instance of Visual Studio using Autocad slnf)

## Checklist:

- [ ] My pull request follows the guidelines in the [Contributing guide](https://github.com/specklesystems/speckle-server/blob/main/CONTRIBUTING.md)?
- [ ] My pull request does not duplicate any other open [Pull Requests](../../pulls) for the same update/change?
- [ ] My commits are related to the pull request and do not amend unrelated code or documentation.
- [ ] My code follows a similar style to existing code.
- [ ] I have added appropriate tests.
- [ ] I have updated or added relevant documentation.

### CODE REVIEW

File: Objects/Converters/ConverterAutocadCivil/ConverterAdvanceSteel2023/ConverterAdvanceSteel2023.csproj [hunk 1]
```diff
@@ -26,7 +26,6 @@
 
 	<Target Name="PostBuild" AfterTargets="PostBuildEvent" Condition="'$(IsDesktopBuild)' == true">
```
Comment (AlanRynne) [inline Objects/Converters/ConverterAutocadCivil/ConverterAdvanceSteel2023/ConverterAdvanceSteel2023.csproj:27]: We've moved away from having a "copy/paste" approach to the post build events in converters to use a specific post build target shared by all projects (it lives in the Directory.Build.targets file at the repo root)

In order to get the converter `dll` copied, you now only need to add a property `<CopyToKitFolder>true</CopyToKitFolder>`

See the revit converter 👇🏼
https://github.com/specklesystems/speckle-sharp/blob/eb68acd457ba3a9d1512baab28d93230ba1ba086/Objects/Converters/ConverterRevit/ConverterRevit2023/ConverterRevit2023.csproj#L11


For any other dependencies, you should specify them in a target that runs after `CopyToKitFolder` does. Revit converter already does this, see here 👇🏼

https://github.com/specklesystems/speckle-sharp/blob/eb68acd457ba3a9d1512baab28d93230ba1ba086/Objects/Converters/ConverterRevit/ConverterRevit2023/ConverterRevit2023.csproj#L32-L36

**Review by AlanRynne — CHANGES_REQUESTED** [review]
Thanks for this @phliberato!

I just have a couple of comments on the post-build events, as the latest cleanup has also done some consolidation on that front, so it's best to align everyone to this.

I posted some links in the comments but if you have any extra questions do let me know ;)

File: Objects/Converters/StructuralUtilities/Triangle/Triangle.csproj [hunk 2]
```diff
@@ -6,4 +6,8 @@
     <RootNamespace>TriangleNet</RootNamespace>
   </PropertyGroup>
 
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
```
[the remark this answers is not in the document]
Comment (phliberato) [reply Objects/Converters/StructuralUtilities/Triangle/Triangle.csproj]: Ok. Now only in AS Connector, it is going.

File: Objects/Converters/ConverterAutocadCivil/ConverterAdvanceSteel2023/ConverterAdvanceSteel2023.csproj (hunk 1 above)
Comment (phliberato) [reply Objects/Converters/ConverterAutocadCivil/ConverterAdvanceSteel2023/ConverterAdvanceSteel2023.csproj]: I did it.

**Review by AlanRynne — APPROVED** [review]
Nice! this looks good!

It's a bit of a busy moment in the repo right now with all the cleanup, thx for bearing with us!

**Pull request merged**